xarray/core/dataarray.py

```diff
@@ -421,6 +422,7 @@ class DataArray(
     _indexes: dict[Hashable, Index]
     _name: Hashable | None
     _variable: Variable
+    _accessors: ClassVar[set[str]] = set()
```

Comment:
Maybe my python knowledge is still not deep enough... But why does this not need to be added to `__slots__`?

Reply:
I'm not entirely certain, but as far as I understand this is because this is a class variable: if we want to use slots on that we'd have to define `__slots__` on the metaclass

Reply:
Yes that was my understanding too @keewis